docs/docs/genai/prompt-version-mgmt/version-tracking/track-application-versions-with-mlflow.mdx

```diff
@@ -60,9 +60,9 @@ Before running the examples below, configure your environment:
     import os
 
     # Set Databricks authentication (only needed when running outside Databricks)
-    os.environ[
-        "DATABRICKS_HOST"
-    ] = "https://your-workspace.databricks.com"  # Your workspace URL
+    os.environ["DATABRICKS_HOST"] = (
+        "https://your-workspace.databricks.com"  # Your workspace URL
+    )
```

Comment:
let's revert unnecessary changes.

Reply:
The lint task was failing, I tried to run it on all files, but it doesn't look like my local version of `blacken-docs` doesn't agree with the CI version. I'm working on it and hopefully fix it soon.

Reply:
Disabled auto-formatting, so only relevant changes are in, and should fix lint error too.